doc/options.rst

```diff
@@ -178,6 +178,8 @@ websearch
 
 * ``--header <FILE.html>`` uses <FILE.html> as header of the search engine web page.
 
+* ``--url <STRING>`` tells the server to accept requests only on URLs using path ``<STRING>`` (for instance ``localhost:8080/<STRING>``). Use ``**`` to accept requests with any path.
```

Comment:
I suggest to name the option to `--restrict-url` which is more informative.
I don't fully understand what this option does actually. Could you clarify the explanation?
In particular the occurrences 2 and 3 of `<STRING>` seems to represent two different variables.
What is the point of `**` if you can get the same without using the option?
Why not interpreting `<STRING>` as  regular expression?
In this case, you could call the option: `--match-url` tells the server to accept requests only on URLs matching `<STRING>` interpreted as a regular expression. Does the url always start with `localhost:8080/`?

Reply:
What about `--accept-url` ?
Another thing not clear: are all the occurrences of <STRING> the same in your explanation of the option?
```
* ``--url <STRING1>`` tells the server to accept requests only on URLs using path ``<STRING2>`` (for instance ``localhost:8080/<STRING3>``). Use ``**`` to accept requests with any path.
```
It seems that we have STRING2=STRING1 but what is the relation between STRING3 and STRING1? Where "localhost:8080/" does it come from?

Reply:
What is the default behavior when we do not use that option?

Reply:
I believe that I now better understand the meaning and all this and propose:
```
* ``--url <STRING>`` tells the server to only accept requests on URLs which path is ``<STRING>``. By default, the server only accepts requests on URLs which path is empty. Use ``**`` to accept requests with any path.
```